static/configeditor/index.html

```diff
@@ -817,6 +817,8 @@ <h4>Software Prerequisites</h4>
     <ul>
      <li>Annotator for Clinical Data Container Edition - Dedicated sandbox instance</li>
      <p>The ACD Configuration Editor requires an installation of ACD Container Edition as a dedicated sandbox instance.  Refer to <a href="https://merative.github.io/acd-containers/installing/installing/">Installing ACD</a> for requirements and installation instructions.  The ACD Configuration Editor needs to be deployed into the same namespace as the dedicated sandbox instance.
```

Comment:
Add a line somewhere that says to use the license use of ConfigurationEditor for the acd dedicated sandbox instance when deploying it.

Reply:
Thanks Dave. I'll look at that again. Wasn't sure where to put it since we refer back to the main docs for installation, but agree, a comment to use that license type here makes sense. I want to get through a meeting tomorrow that Kristin set up for some questions on the licensing before merging this PR, just in case that causes some additional changes.